### PR TITLE
Damien Invasion!

### PR DESCRIPTION
**Pull request by recursiveforte**
## Describe your project in 1-3 sentences

LED matrix space-invaders style hybrid art piece multiplayer game thing

## Hack Club Slack handle

`@Cheru Berhanu` & `@The Bread Demon`

## Checklist before submitting proposal:

- [x] I am a current high school student (or younger).
- [x] I have submitted an RSVP at <https://hackclub.com/winter>
- [x] I am a member of the Hack Club Slack (Join at <https://hackclub.com/slack>)
- [x] I have self-reviewed my project proposal.

### CODE REVIEW

**Review by bellesea — CHANGES_REQUESTED** [review]
hii @recursiveforte omg this is a super cool idea!! i'd love to try it out haha :)

could you please have your friend also comment on this PR confirming:
- [ ] I am a current high school student (or younger).
- [ ] I have submitted an RSVP at <https://hackclub.com/winter>
- [ ] I am a member of the Hack Club Slack (Join at <https://hackclub.com/slack>)
- [ ] I have self-reviewed my project proposal.

a few questions:
- are you referencing any guides for this?
- will you be designing your own enclosure or using an online design?
- how do you plan on controlling 4 LED Matrixes with one bonnet?
- how will you be powering the LED Matrixes?
- your assembled matrix would be smaller than the canvas, is that intentional?

thanks!

**Comment by recursiveforte** [comment]
I'll ask them! thanks!

1) no, not outside of the basic guides for how to use matrix panels
2) we'll be probably designing our own enclosure
3) the matrixes can be changed, only needing one pi and one bonnet
4) we'll be using the four 10A power supplies in the budget (seems like a lot, but it looks like we'll need it)
5) yes, because the internal area of the canvas (minus the wood edges) will probably be a lot smaller than 20in^2

**Comment by TheBreadDemon** [comment]
I’m the other person working on the project! I am indeed in high school (my friend and I are in the same grade), I’ve applied for the winter RSVP, and I’ve reviewed the project proposal. Thanks!

File: recursiveforte.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+---
+name: "@recursiveforte"
```
Comment (bellesea) [inline recursiveforte.md:2]:
```suggestion
name: ["@recursiveforte", "@TheBreadDemon"]
```

**Review by bellesea — APPROVED** [review]
@recursiveforte and @TheBreadDemon  thank you sm for elaborating! 

an awesome member of @hackclub/bank-ops will be reaching out soon with next steps :)

**Review by hackliv — APPROVED** [review]
Hey @recursiveforte and @TheBreadDemon this sounds so cool! I'd love to meet you both and hear about the inspo behind this!!  I'm dming you both now on Slack to chat about next steps. 😁

**Pull request merged**